composeApp/src/commonMain/kotlin/romHandlers/abstractRomHandlers/AbstractDSRomHandler.kt

```diff
@@ -0,0 +1,15 @@
+package romHandlers.abstractRomHandlers
+
+import configurations.RomConfiguration
+import models.NDSFile
+import models.NDSRom
+import models.Rom
+
+abstract class AbstractDSRomHandler(romConfiguration: RomConfiguration,
+                                    protected var rom: NDSFile)
+    : AbstractRomHandler(romConfiguration)
+{
+    override fun saveROM(): Rom {
```

Comment:
```suggestion
    override fun saveROM(): NDSRom {
```
That should be a valid override